packages/ckeditor5-utils/src/config.ts

```diff
@@ -238,23 +265,31 @@ export default class Config<Cfg> {
 }
 
 // Clones configuration object or value.
-// @param {*} source Source configuration
-// @returns {*} Cloned configuration value.
+// @param source Source configuration
+// @returns Cloned configuration value.
 function cloneConfig<T>( source: T ): T {
 	return cloneDeepWith( source, leaveDOMReferences );
 }
 
 // A customized function for cloneDeepWith.
 // It will leave references to DOM Elements instead of cloning them.
 //
-// @param {*} value
-// @returns {Element|undefined}
+// @param value
 function leaveDOMReferences( value: unknown ): unknown {
 	return isElement( value ) ? value : undefined;
 }
 
+/**
```

Comment:
Block comments describing internal things are not visible in the `output.json` file. We should use inline comments or export these types.